src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -1026,6 +1026,173 @@ end finite_measure --namespace
 
 end normalize_finite_measure -- section
 
+section limsup_closed_le_and_le_liminf_open
```

Comment:
I think this section would be worth a mention in the file-level docstring.

Reply:
I will add a bullet point to what is currently a TODO about the portmanteau theorem. Once the rest of the implications are there, I plan to rewrite the docstring into a more comprehensive description of how these characterizations of weak convergence are meant to be used.